code/datums/status_effects/debuffs.dm

```diff
@@ -297,6 +298,89 @@
 	owner.underlays -= marked_underlay //if this is being called, we should have an owner at this point.
 	..()
 
+/datum/status_effect/eldritch
+	duration = 15 SECONDS
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	///underlay used to indicate that someone is marked
+	var/mutable_appearance/marked_underlay
+	///path for the underlay
+	var/effect_sprite = ""
+
+/datum/status_effect/eldritch/on_apply()
+	if(owner.mob_size >= MOB_SIZE_HUMAN)
+		marked_underlay = mutable_appearance('icons/effects/effects.dmi', effect_sprite)
+		marked_underlay.pixel_x = -owner.pixel_x
+		marked_underlay.pixel_y = -owner.pixel_y
+		owner.underlays += marked_underlay
```

Comment:
Use overlays with proper layer instead, and register to signal instead of adding it directly.

Reply:
Signals didnt work. i defaulted to use add_overlay and cut_overlay.